### ISSUE TITLE
showstopper: "cannot start the seamless server"

### ISSUE THREAD

**Issue by karlkleinpaste** [Xpra-org/xpra#3663]
Ubuntu 20.04; just upgraded to v5.0-r32148 (g18720c1f1) beta.

`xpra start :73 --attach=no --start-child=vterm`

`vterm` is a wrapper script that mostly starts a terminal.
`/run/user/45512/xpra/27/server.log` in its entirety:
```
2022-10-21 10:37:43,021 [:73]no uinput module (not usually needed)
2022-10-21 10:37:44,603 [:73]pointer device emulation using XTest
2022-10-21 10:37:44,619 [:73]serving html content from '/usr/share/xpra/www'
2022-10-21 10:37:44,619 [:73]Error: cannot start the seamless server
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/xpra/scripts/server.py", line 1473, in _do_run_server
    app.init(opts)
  File "/usr/lib/python3/dist-packages/xpra/x11/server.py", line 96, in init
    super().init(opts)
  File "/usr/lib/python3/dist-packages/xpra/x11/x11_server_core.py", line 101, in init
    super().init(opts)
  File "/usr/lib/python3/dist-packages/xpra/server/server_base.py", line 155, in init
    c.init(self, opts)
  File "/usr/lib/python3/dist-packages/xpra/server/server_core.py", line 272, in init
    self.init_auth(opts)
  File "/usr/lib/python3/dist-packages/xpra/server/server_core.py", line 687, in init_auth
    opts_value = getattr(opts, f"{x}_auth")
AttributeError: 'Values' object has no attribute 'socket_auth'
2022-10-21 10:37:44,620 [:73]'Values' object has no attribute 'socket_auth'
2022-10-21 10:37:44,620 [:73]
```

**Comment by totaam** [comment]
I am quite confident that this has already been fixed: 8e1c2a3e3b6a5969bc1b141c6b82b100874d130a

**Comment by karlkleinpaste** [comment]
I had updated from beta repo (`apt-get update` etc) 15min before I wrote the report. Perhaps this means an updated build for the repos is in order.
